ztp/source-crs/StorageLVMOSubscriptionNS.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-storage
+  labels:
+    openshift.io/cluster-monitoring: "true"
+  annotations:
+    workload.openshift.io/allowed: management
```

Comment:
We need to remove this annotation. The LVMO isn't part of the management partition.